src/metrics.ts

```diff
@@ -218,6 +225,25 @@ export const MetricsProcessor = (
     eventBus.emit(MetricEvent.READY);
   };
 
+  const valueToString = (value: any): string => {
+    switch (typeof value) {
+      case 'string':
+        return value;
+      case 'number':
+      case 'boolean':
+      case 'bigint':
+      case 'symbol':
+        return value.toString();
+      case 'object':
+        if (Array.isArray(value)) {
+          return value.toString()
+        }
+        return value === null ? '' : JSON.stringify(value);
+      default:
+        return String(value);
+    }
```

Comment:
I think you can probably simplify this by just leaning on `String` and checking for 
```suggestion
    return typeof value === 'object' && !Array.isArray(value) ? JSON.stringify(value) : String(value)
```

Reply:
I very much like this. I tried to simplify it just before your comment. Would you mind refreshing your page and making the suggestion again, and I'll commit it!

Reply:
Or I'll just make the change myself

Reply:
Done - tested with array, object and !== object. Works well.

Reply:
`null` would throw it off since `null` will return `object` as its type, but you're filtered them out already. Good eye for detail BTW including `Symbol` and `BigInt`, about 99.999% of devs don't even know they exist!